### PR TITLE
chore: update k8s modules to 1.19 final version

### PR DESCRIPTION
**Pull request by smira**
I think we missed that when updating K8s for the final version.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/2467)

### CODE REVIEW

**Comment by rsmitty** [comment]
Ugh, my fault. Thanks!

File: go.mod [hunk 1]
```diff
@@ -64,7 +64,7 @@ require (
 	github.com/u-root/u-root v6.0.0+incompatible // indirect
 	github.com/vishvananda/netns v0.0.0-20200520041808-52d707b772fe // indirect
 	github.com/vmware/vmw-guestinfo v0.0.0-20200218095840-687661b8bd8e
-	go.etcd.io/etcd v0.5.0-alpha.5.0.20200716221620-18dfb9cca345 // v3.4.10
+	go.etcd.io/etcd v0.5.0-alpha.5.0.20200819165624-17cef6e3e9d5 // v3.4.10
```
Comment (smira) [inline go.mod:67]: this is something which was updated behind the scenes

Comment (smira) [reply go.mod:67]: but it seems to be pulled in by Kubernetes, might cause merge conflict in 0.6

**Pull request merged**